main.py

```diff
@@ -2,26 +2,60 @@
 import models
 import datetime
 
+
 def add_heart_rate(email, heart_rate, time):
-    user = models.User.objects.raw({"_id": "suyash@suyashkumar.com"}).first() # Get the first user where _id=suyash@suyashkumar.com
-    user.heart_rate.append(heart_rate) # Append the heart_rate to the user's list of heart rates
-    user.heart_rate_times.append(time) # append the current time to the user's list of heart rate times
-    user.save() # save the user to the database
+    user = models.User.objects.raw({"_id": email}).first()
+    user.heart_rate.append(heart_rate)
+    user.heart_rate_times.append(time)
+    user.save()
+
+
+def create_user(email, age=None, hr=None):
+    u = models.User(email, age, [], [])
+    if hr:
```

Comment:
if `hr` is 0 this will be false-y and will not trigger. probably best to use `if hr is not None`